README.md

```diff
@@ -7,13 +7,13 @@
 [![codecov](https://codecov.io/gh/containerbase/base/branch/main/graph/badge.svg?token=GYS2ZZAXDP)](https://codecov.io/gh/containerbase/base)
 
 This repository is the source for the Docker images [`containerbase/base`](https://hub.docker.com/r/containerbase/base) and `ghcr.io/containerbase/base`.
-Commits to `main` branch are automatically build and published.
+The commits to the `main` branch are automatically built and published.
 
 ## Local development
 
-You need a recent [docker](https://www.docker.com) version with [buildx](https://github.com/docker/buildx) [`>= v0.4.0`](https://github.com/docker/buildx/releases/tag/v0.4.0) plugin installed.
+You need a recent [Docker](https://www.docker.com) version and install the [`buildx`](https://github.com/docker/buildx) [`>= v0.4.0`](https://github.com/docker/buildx/releases/tag/v0.4.0) plugin.
```

Comment:
```suggestion
You need a recent [Docker](https://www.docker.com) version and install the latest [`buildx`](https://github.com/docker/buildx) plugin.
```
we should link to the latest release. maybe we already need a higher version. 🤔

let's ensure latest version